pennylane/tape/tapes/tape.py

```diff
@@ -40,6 +40,8 @@
     qml.GaussianState,
 )
 
+FORBIDDEN_ON_SAME_WIRE = [qml.operation.Expectation, qml.operation.Variance]
```

Comment:
How come `qml.sample()` is not forbidden on the same wire?

Reply:
Good point, I'll add that!